### PR TITLE
[processing] fix #2396 multi ring buffer

### LINKED ISSUE #2396: [processing][FEATURE] multiring buffer algorithm
Original commit: https://github.com/qgis/QGIS/commit/04ce43569e6a0c9bb6238cbb8a7c93fe3389bed2 by alexbruy

Unfortunately this naughty coder did not write a description... :-(

### PR DESCRIPTION
**Pull request by ghtmtt**
should fix #2396

### CODE REVIEW

File: source/docs/user_manual/processing_algs/qgis/vectorgeometry.rst [hunk 1]
```diff
@@ -1144,6 +1144,39 @@ See also
...
+   :align: center
+
+Parameters
+..........
+
+``Input layer`` [vector: any]
+  Input vector layer.
+
+``Number of rings`` [number]
+  Total number of rings that the buffer must have.
```
Comment (DelazJ) [inline source/docs/user_manual/processing_algs/qgis/vectorgeometry.rst]: Could we mention that the option can be unique for all features or based on feature's data? (same for the other option)

**Comment by ghtmtt** [comment]
@DelazJ description added for both parameters based on data defined values. If we are ok with this I can merge it

Thanks for the review!

File: source/docs/user_manual/processing_algs/qgis/vectorgeometry.rst [hunk 2]
```diff
@@ -1144,6 +1141,46 @@ See also
...
+   Multi-ring buffer for line, point and polygon layer
+
+Parameters
+..........
+
+``Input layer`` [vector: any]
+  Input vector layer.
+
+``Number of rings`` [number]
+  Total number of rings that the buffer must have. It can be an unique value (same
```
Comment (DelazJ) [inline source/docs/user_manual/processing_algs/qgis/vectorgeometry.rst]: a unique

File: source/docs/user_manual/processing_algs/qgis/vectorgeometry.rst [hunk 3]
```diff
@@ -1144,6 +1141,46 @@ See also
...
+  ring number for all the features) or it can be taken from features data (different
+  ring number depending on the feature values).
+
+``Distance between rings`` [number]
+  Distance between the single rings. It can be an unique value (same distance for
```
Comment (DelazJ) [inline source/docs/user_manual/processing_algs/qgis/vectorgeometry.rst]: a unique

File: source/docs/user_manual/processing_algs/qgis/vectorgeometry.rst [hunk 4]
```diff
@@ -1144,6 +1141,46 @@ See also
...
+  all the the features) or it can be taken from features data (different distance
```
Comment (DelazJ) [inline source/docs/user_manual/processing_algs/qgis/vectorgeometry.rst]: the the

**Pull request merged**